### PR TITLE
{vis}[GCCcore/6.4.0] fontconfig v2.12.1

### PR DESCRIPTION
**Pull request by JackPerdue**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/e/expat/expat-2.2.1-GCCcore-6.4.0.eb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+easyblock = 'ConfigureMake'
+
+name = 'expat'
+version = '2.2.1'
```
Comment (boegel) [inline easybuild/easyconfigs/e/expat/expat-2.2.1-GCCcore-6.4.0.eb]: @JackPerdue We already have an easyconfig for `expat` 2.2.5, including this old version doesn't make sense?

File: easybuild/easyconfigs/f/fontconfig/fontconfig-2.12.1-GCCcore-6.4.0.eb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+source_urls = ['http://www.freedesktop.org/software/fontconfig/release/']
+sources = [SOURCE_TAR_GZ]
+checksums = ['a9f42d03949f948a3a4f762287dbc16e53a927c91a07ee64207ebd90a9e5e292']
+
+builddependencies = [
+    ('binutils', '2.28'),
+]
+
+dependencies = [
+    ('expat', '2.2.1'),
```
Comment (boegel) [inline easybuild/easyconfigs/f/fontconfig/fontconfig-2.12.1-GCCcore-6.4.0.eb]: @JackPerdue bump to 2.2.5?

File: easybuild/easyconfigs/l/libpng/libpng-1.6.30-GCCcore-6.4.0.eb [hunk 3]
```diff
@@ -0,0 +1,35 @@
+easyblock = 'ConfigureMake'
+
+name = 'libpng'
+version = '1.6.30'
```
Comment (boegel) [inline easybuild/easyconfigs/l/libpng/libpng-1.6.30-GCCcore-6.4.0.eb]: @JackPerdue We already have an easyconfig for `libpng` 1.6.34, there's no point in adding one for an older version?

**Comment by JackPerdue** [comment]
oops... old... closing... see #6224

**Pull request closed without merging**